docs/content/usage/configuration.md

```diff
@@ -82,7 +82,7 @@ template:
     # Configures whether to tabulate variables in the output. When set to `True`,
     # variables will be displayed in a tabular format intsead of plain marktdown sections.
     # NOTE: This option does not support rendering multiline code blocks.
-    tabulate_vars: False
+    tabulate_defaults: False
```

Comment:
IMO, a dedicated setting tabulation of variables is a bit overkill. If someone wants to use a table to display vars it should be used for defaults and vars.